lerna.json

```diff
@@ -1,5 +1,5 @@
 {
-  "version": "0.2.0",
+  "version": "0.3.0",
```

Comment:
Should this be 0.4.0 since thats the next release?

Reply:
This is just me playing catchup since we didn't update the versions via lerna last time. We'll want to bump them up on the release branch for the coming one.